app/controllers/user_controller.rb

```diff
@@ -38,6 +38,18 @@ def show
     end
   end
 
+  def update_profile_image
+    current_user = User.find_by(:id => params[:id])
+    upload_image = Cloudinary::Uploader.upload(params["image_url"])
+    user = { image_url: upload_image["url"] }
+    begin
+      current_user.update!(user)
+      render json: { url: upload_image["url"] }, status: :ok
+    rescue => exception
+      render json: exception, status: :bad_request
+    end
```

Comment:
You can handle exceptions more generically in Rails by adding a [rescue_from](https://api.rubyonrails.org/classes/ActiveSupport/Rescuable/ClassMethods.html#method-i-rescue_from) in the `ApplicationController`. For example,

```rb
rescue_from ActiveRecord::RecordInvalid, with: :render_bad_request

def render_bad_request(exception)
  render json: exception, status: :bad_request
end
```

Exceptions should be a bit more specific so that you still allow for true errors to show up.